tools/RAiDER/losreader.py

```diff
@@ -462,7 +379,89 @@ def read_ESA_Orbit_file(filename, ref_time):
     return [t, x, y, z, vx, vy, vz]
 
 
-# @jit(nopython=True)
+############################
+def state_to_los(svs, xyz_targets):
```

Comment:
Somewhere in this vicinity or replacing this function is where I envisioned isce3 getting called, though I'm not 100%. One complexity is that it looked to me like  isce3 currently only handles rasters and points instead of ndarrays, which is what gets passed here, is that true? If so that will require looping over the points in python, which is actually what was originally done using geo2rdr. That introduces a large time overhead, though, so would be ideal not to do so.

Reply:
I can take a shot at it. I suspect making a 1D array like a `1xN` array would work out of the box.